tests/qos/test_qos_sai.py

```diff
@@ -1126,6 +1127,8 @@ def testQosSaiLossyQueueVoq(
             Raises:
                 RunAnsibleModuleFail if ptf test fails
         """
+        pytest_require(separated_dscp_to_tc_map_on_uplink(dut_qos_maps_module),
```

Comment:
encounter below error when run testQosSaiLossyQueueVoq testcase:

>       pytest_require(separated_dscp_to_tc_map_on_uplink(dut_qos_maps_module),
                       "Skip test because separated QoS map is not applied")
 
LossyVoq   = 'lossy_queue_voq_1'
dutConfig  = {'dstDutInstance': <MultiAsicSonicHost bjw-can-8102-1>, 'dualTor': False, 'dualTorScenario': True, 'dutAsic': 'gb', ...}
dutQosConfig = {'param': {'100000_120000m': {'lossy_queue_1': {'cell_size': 384, 'dscp': 8, 'ecn': 1, 'packet_size': 1350, ...}, 'los...': {'dscp': 4, 'ecn': 1, 'outer_dscp': 6, 'packet_size': 1350, ...}, ...}, ...}, 'portSpeedCableLength': '100000_300m'}
dutTestParams = {'basicParams': {'downlink_port_ids': [34, 36, 37, 38, 39, 42, ...], 'downlink_port_ips': ['10.0.0.33', '10.0.0.35', '...net152', 'Ethernet156', 'Ethernet168', ...], 'dst_asic_index': 0, ...}, 'hwsku': 'Cisco-8102-C64', 'topo': 't1-64-lag'}
duthost    = <MultiAsicSonicHost bjw-can-8102-1>
get_src_dst_asic_and_duts = {'all_asics': [<SonicAsic 0>], 'all_duts': [<MultiAsicSonicHost bjw-can-8102-1>], 'dst_asic': <SonicAsic 0>, 'dst_asic_index': 0, ...}
ingressLossyProfile = {'bufferPoolRoid': '0x1800000000000001', 'bufferPoolVoid': 'oid:0x18000000000965', 'dynamic_th': '7', 'pool': 'ingress_lossless_pool', ...}
localhost  = <tests.common.devices.local.Localhost object at 0x7f501adecf40>
ptfhost    = <tests.common.devices.ptf.PTFHost object at 0x7f501d042c40>
self       = <tests.qos.test_qos_sai.TestQosSai object at 0x7f501ca93fa0>
 
qos/test_qos_sai.py:1130:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
 
dut_qos_maps_module = <function dut_qos_maps_module at 0x7f501dd9fb80>
 
    def separated_dscp_to_tc_map_on_uplink(dut_qos_maps_module):
        """
        A helper function to check if separated DSCP_TO_TC_MAP is applied to
        downlink/unlink ports.
        """
        dscp_to_tc_map_names = set()
>       for port_name, qos_map in dut_qos_maps_module['port_qos_map'].items():
E       TypeError: 'function' object is not subscriptable

Reply:
@XuChen-MSFT : I think we don't need this pytest_require at all. We should be able to run the test if the seperated map is present or not. I am removing this line completely.